website/docs/user-guides/create-resilience-probe.md

```diff
@@ -41,3 +47,25 @@ Configure the details for the probe you are creating, once completed, click the
 The new probe will appear in the list as shown:
 
 <img src={require('../assets/user-guides/resilience-probes/create-probe/step-6.png').default} />
+
+### Experiment Creation
+
+When creating an experiment, it's imperative to include the Resilience Probe as part of the setup. This step is now mandatory to ensure accurate chaos injection and monitoring during the experiment. Follow these steps to add the probe to the experiment configuration:
+
+1. **Identify Chaos Injection Points**: Determine the points within your system where chaos will be injected.
+
+2. **Select Resilience Probe**: Choose the appropriate Resilience Probe that aligns with your experimentation goals and the type of chaos you want to inject as discussed above.
+
+3. **Integrate Probe into Experiment YAML**: Add the Resilience Probe configuration to your experiment YAML file. Ensure that the probe is properly configured and referenced within the experiment setup.
+
+4. **Validate Experiment Configuration**: Before initiating the experiment, validate the experiment configuration to ensure that the Resilience Probe is correctly included and configured.
```

Comment:
We don't need these changes here, rather in the [Schedule a Chaos Experiment](https://docs.litmuschaos.io/docs/user-guides/schedule-experiment) page we need to add one more step for adding a probe to a fault. 

We can just add a note here that at least one resilience probe needs to be added to each fault now.

Reply:
I have added the note for the same you have mentioned to be in create-resilience-probe.md
For further changes in [Schedule a Chaos Experiment](https://docs.litmuschaos.io/docs/user-guides/schedule-experiment),Should I do these changes in the same PR or opened a new issue?

Reply:
We can take the changes in this PR itself.